charts/osm/templates/osm-deployment.yaml

```diff
@@ -16,8 +16,14 @@ spec:
         app: osm-controller
       annotations:
           checksum/config: {{ print .Values.openServiceMesh | sha256sum }}
+          seccomp.security.alpha.kubernetes.io/pod: runtime/default
```

Comment:
@ritazh  FYI - runtime/default is not supported in OpenShift. Arc team mentioned they support a helm variable for K8s distro and we can use that with helm if else construct to specify the values accordingly in the chart.

Reply:
Thanks @RamyasreeChakka! Do you recall what the variable is so we can add it?

Reply:
Please check with Arc team. Not sure if it is currently available but it is in their plans.

Reply:
Added this https://github.com/Azure/osm-azure/issues/17 to track in the future.